debian/control

```diff
@@ -0,0 +1,13 @@
+Source: python-ev3dev
+Maintainer: Denis Demidov <dennis.demidov@gmail.com>
+Section: python
+Priority: optional
+Build-Depends: python-setuptools (>= 0.6b3), python-all-dev (>= 2.6.6-3), debhelper (>= 7)
+Standards-Version: 3.9.5
+Homepage: http://www.ev3dev.org/
+
+Package: python-python-ev3dev
```

Comment:
does this conform with <https://www.debian.org/doc/packaging-manuals/python-policy/ch-module_packages.html#s-package_names>?